infrastructure/src/main/java/org/corfudb/infrastructure/management/PollReport.java

```diff
@@ -45,6 +47,7 @@
     /**
      * List of responsive servers in a current layout
      */
+    @Deprecated
```

Comment:
Can we remove this if not required?
Is this used in a lot of places?

Reply:
Yep.

Reply:
No longer relevant.